model/qti/ParserFactory.php

```diff
@@ -1528,16 +1528,14 @@ protected function getNonEmptyChildren(DOMElement $data)
         return $returnValue;
     }
 
-    private function buildStylesheet(DOMElement $data)
+    private function buildStylesheet(DOMElement $data): Stylesheet
     {
-        $returnValue = new Stylesheet([
-            'href' => (string) $data->getAttribute('href'),
-            'title' => $data->hasAttribute('title') ? (string) $data->getAttribute('title') : '',
-            'media' => $data->hasAttribute('media') ? (string) $data->getAttribute('media') : 'screen',
-            'type' => $data->hasAttribute('type') ? (string) $data->getAttribute('type') : 'text/css',
+        return new Stylesheet([
+            'href' => $data->getAttribute('href'),
+            'type' => $data->hasAttribute('type') ? $data->getAttribute('type') : 'text/css',
```

Comment:
I know the interface says it return string, but why we had casting before. Just double checking if it is really safe to remove :)

Reply:
I will check it 😄